tests/api2/test_bootenv.py

```diff
@@ -1,24 +1,68 @@
+from functions import wait_on_job
 from middlewared.test.integration.utils import call, ssh
 
+def test_get_default_environment_and_make_new_one():
+    active_be_id = call('bootenv.query', [['activated', '=', True]], {'get': True})['id']
+
+    # create duplicate name to test failure
+    results = call('bootenv.create', {'name': active_be_id, 'source': active_be_id})
+    assert results == f'[EEXIST] bootenv_create.name: The name "{active_be_id}" already exists'
+
+    # create new bootenv and activate it
+    call('bootenv.create', {'name': 'bootenv01', 'source': active_be_id})
+    assert len(call('bootenv.query', [['name', '=', 'bootenv01']]).json()) == 1
```

Comment:
This call to `.json()` is wrong.

Reply:
Good catch, I swear I looked for all of those lol